src/bp/ui-studio/src/web/components/App/index.jsx

```diff
@@ -54,6 +55,16 @@ class App extends Component {
     EventBus.default.on('hints.updated', () => {
       this.props.refreshHints()
     })
+
+    EventBus.default.on('logs::' + window.BOT_ID, ({ id, level, message, args }) => {
```

Comment:
Just wondering, if all logs are streamed, doesn't it means that someone could change window.BOT_ID to * or to another bot where he doesn't have access ?

Reply:
Yes currently the socket is authenticated but it's a global authentication, i.e. no per-bot subscription.  That's always been the case so decided to leave it as-is. 

For future im, the concept of [`__room`](https://github.com/botpress/botpress/blob/a2cc21dd278546362621466e1a7133b8f73fc3d1/src/bp/core/sdk/impl.ts#L25) can be used